site/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>amb</title>
+  <link href="../assets/styles/normalize.css" rel="stylesheet" >
+  <link href="../assets/styles/main.css" rel="stylesheet" >
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <div>
+        <h3>amb</h3>
```

Comment:
<h3> element should be on the outside of the <a> tag. It shouldn't need a div around it.